renku/core/management/storage.py

```diff
@@ -83,17 +83,19 @@ def has_external_storage(self):
         :raises: ``errors.ExternalStorageNotInstalled``
         :raises: ``errors.ExternalStorageDisabled``
         """
+        if not self.use_external_storage:
```

Comment:
Isn't this unrelated to whether the repository has external storage enabled? I think they should be kept as two separate checks.

Reply:
I am not sure if it was the intention of this property to make them separated because `use_external_storage` was checked here before:
```
if self.use_external_storage and not storage_enabled:
    raise errors.ExternalStorageDisabled(self.repo)
```
Moreover, it does not make sense to complain about LFS not installed when user explicitly asked to not use LFS.

Reply:
it does if it's necessary to pull LFS objects in order to re-execute something

Reply:
OK.